crates/cdk/src/nuts/nut05.rs

```diff
@@ -264,6 +266,8 @@ pub struct MeltMethodSettings {
     /// Max Amount
     #[serde(skip_serializing_if = "Option::is_none")]
     pub max_amount: Option<Amount>,
+    /// Amountless
+    pub amount_less: bool,
```

Comment:
```suggestion
    pub amountless: Option<bool>,
```

The NUTs PR has this without an underscore and it can be null so it much match here.